scripts/cnv_cromwell_tests/somatic/cnv_somatic_pair_wes_do-gc_workflow.json

```diff
@@ -1,6 +1,7 @@
 {
   "CNVSomaticPairWorkflow.common_sites": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/cnv_somatic_workflows_test_files/common_snps_sample-chr20.interval_list",
   "CNVSomaticPairWorkflow.gatk_docker": "__GATK_DOCKER__",
+  "CNVSomaticPairWorkflow.PreprocessIntervals.bin_length": 0,
```

Comment:
Note that it must have been the case that this doesn't actually affect the results for this test data, otherwise you would've had to update the PoN test file.

This must be because there is only one target above the default length of 1000, and its corresponding bins must've gotten filtered out.

In any case, it's probably good that you added this here to guard against future changes to the test data.

Reply:
However, one thing---could you move the line to after that for `normal_bam_idx`, to maintain the alphabetical `sort` order?